docs/maya/simshape.md

```diff
@@ -200,12 +203,14 @@ To enable the debugger the *Debug* checkbox must be marked. To select the specif
 
  - **Collision Constraints**: For each vertex, a line will be drawn from the mesh to the closest point of a collider. The debug lines will only be displayed in case collisions are enabled and colliders have been set up.
  - **Muscle Fibers**: For each vertex, a line will be drawn showing the direction of the muscle fibers. The debug lines will only be displayed in case muscle activations have been enabled with an AdonisFX Muscle Patches file.
+ - **Sliding Surface On Collider**: For each vertex, lines will outline the collider triangles within the reach of its *Sliding Distance*.
```

Comment:
Should we refer to the sliding distance as "Max Sliding Distance"? Same question applies to skin.

Reply:
Yes! Applied